project_event/models/project_task.py

```diff
@@ -378,14 +378,26 @@ def get_message_body(self, action):
         return switcher.get(action)
 
     def get_message(self, action):
+        message = '<br>'
+        responsible = self.responsible_id.id
+        if self.activity_task_type == 'activity':
+            message += _('Avtivity: <br>') + self.name + '<br>'
+            message += _('Tasks: <br>')
+            for indx, task in enumerate(self.child_ids):
+                message += task.name
+                if indx < len(self.child_ids)-1:
```

Comment:
@eilst  for indx, task in enumerate(self.child_ids): "indx" change something meaningful like index_task, or ....

Reply:
OK, right